arduino/netsender/NetSender.cpp

```diff
@@ -25,20 +25,43 @@
 #include <limits.h>
 #include <string.h>
 #include <ctype.h>
+
+#define ESP32
+
+#ifdef ESP8266
 #include <ESP8266WiFi.h>
 #include <ESP8266HTTPClient.h>
 #include <WiFiClient.h>
 #include <EEPROM.h>
-//#include "nonarduino.h"
+#endif
+#ifdef ESP32
+#include <WiFi.h>
+#include <HTTPClient.h>
+#include <EEPROM.h>
```

Comment:
This is probably something that could happen later, but apparently ```EEPROM.h``` is deprecated for the ESP32 with a preference for ```preferences.h``` which has the following advantages 

1. No need to call ```EEPROM.commit();``` data is saved automatically.
2. Key-value based storage.
3. Has wear levelling to reduce wear and tear on flash.

Reply:
Yes, a separate PR.